pkgs/stdenv/generic/setup.sh

```diff
@@ -116,13 +118,14 @@ exitHandler() {
 
     if [ -n "${showBuildStats:-}" ]; then
         times > "$NIX_BUILD_TOP/.times"
-        local -a times=($(cat "$NIX_BUILD_TOP/.times"))
+        local -a buildTimesArray
+        IFS=" " read -r -a buildTimesArray <<< "$(cat "$NIX_BUILD_TOP/.times")"
```

Comment:
```suggestion
        IFS=" " read -r -a buildTimesArray < "$NIX_BUILD_TOP/.times"
```
`cat` is generally superfluous and can be replaced with `<`

Reply:
It sounds right, let me test this.